atmo/templates/atmo/dashboard.html

```diff
@@ -83,11 +83,12 @@
           <th>Job timeout</th>
           <th>Start date</th>
           <th>End date</th>
+          <th>Status</th>
         </tr>
       </thead>
       <tbody>
         {% for spark_job in spark_jobs %}
-        <tr>
+        <tr class="{{ spark_job.latest_run.status_css_class }}">
```

Comment:
I'm not a fan of using colored rows for this since it can at worst become a rainbow like table that is hard to read. Instead let's use bold, colored text for the status text maybe a descriptive icon from the [Glyphicons](http://getbootstrap.com/components/#glyphicons) set bootstrap provides. E.g. "glyphicon glyphicon-ok" for succeess, "glyphicon glyphicon-remove" for error?